x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -802,6 +810,110 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     return this.esClient.search<SafeEndpointEvent>(query);
   }
 
+  public async fetchValueListMetaData(interval: number) {
+    if (this.esClient === undefined || this.esClient === null) {
+      throw Error('elasticsearch client is unavailable: cannot retrieve diagnostic alerts');
+    }
+
+    const listQuery: SearchRequest = {
+      expand_wildcards: ['open' as const, 'hidden' as const],
+      index: '.lists-*',
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      body: {
+        aggs: {
+          total_value_list_count: {
+            cardinality: {
+              field: 'name',
+            },
+          },
+          type_breakdown: {
+            terms: {
+              field: 'type',
```

Comment:
While we only expose 4 list types in the UI, the [API allows creation of ~20 types of value lists](https://github.com/elastic/kibana/blob/8.4/packages/kbn-securitysolution-io-ts-list-types/src/common/type/index.ts#L15) right now. Retrieving as many buckets here as the number of list types would ensure that we get accurate type breakdowns across all types.

Reply:
Added size of 50 to the terms aggregation to include all types and to be safe(not sure if more types would be added in future)